src/dissect/dissect.c

```diff
@@ -2214,8 +2232,8 @@ static int run(int argc, char *argv[]) {
                         if (arg_in_memory)
                                 return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "--in-memory= not supported when operating via systemd-mountfsd.");
 
-                        if (arg_loop_ref)
-                                return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "--loop-ref= not supported when operating via systemd-mountfsd.");
+                        if (arg_loop_ref || arg_loop_ref_auto)
```

Comment:
Strictly speaking, `|| arg_loop_ref_auto` is redundant, as `arg_loop_ref` is anyway set unconditionally. But I think it is the current form is better as the condition is consistent to the log message.